README.md

```diff
@@ -90,13 +104,22 @@ Every route handler receives exactly one parameter: a context object. You should
 - `redisClient`: a [node-redis](https://github.com/NodeRedis/node-redis) client; present if you have enabled the redis feature
 - `postgresClient`: a [node-postgres](https://github.com/brianc/node-postgres) client; present if you have enabled the postgresql feature
 
+### Route handlers
+
+Route handlers are functions with a `route` field. The route field is a [find-my-way](https://github.com/delvedor/find-my-way) route with a little sugar. The handler is a function that takes a context object and returns a plain old Javascript object *or* throws an error.
 
+Boltzmann looks for handlers exported by the following two places:
 
-### Route handlers
+- from `handlers.js` in the same directory (good for small services)
+- from a `handlers/` directory
+
+The scaffolding tool gives you a working `handlers.js` file.
+
+The route field should look like this: `VERB /path/with/:params`. The first segment of the string must be an http verb that `find-my-way` can pass to its `route.on()`. The second segment must be a path format that `find-my-way` understands.
```

Comment:
We should weasel this a bit at the end with "If you need to accept 2 or more verbs, you can configure the verbs and the path separately. Here's a link to a guide."